WSRequestHandler.cpp

```diff
@@ -1343,6 +1344,156 @@ void WSRequestHandler::HandleSetSceneItemCrop(WSRequestHandler* req) {
 }
 
 /**
+ * Gets the scene specific properties of the specified source item.
+ *
+ * @param {String (optional)} `scene-name` the name of the scene that the source item belongs to. Defaults to the current scene.
+ * @param {String} `item` The name of the source.
+ *
+ * @return {String} `name` The name of the source.
+ * @return {int} `position.x` The x position of the source from the left.
+ * @return {int} `position.y` The y position of the source from the top.
+ * @return {double} `rotation` The clockwise rotation of the item in degrees around the point of alignment.
+ * @return {int?string?} `alignment` The point on the source that the item is manipulated from.
+ * @return {double} `scale.x` The x-scale factor of the source.
+ * @return {double} `scale.y` The y-scale factor of the source.
+ * @return {int} `crop.top` The number of pixels cropped off the top of the source before scaling.
+ * @return {int} `crop.right` The number of pixels cropped off the right of the source before scaling.
+ * @return {int} `crop.bottom` The number of pixels cropped off the bottom of the source before scaling.
+ * @return {int} `crop.left` The number of pixels cropped off the left of the source before scaling.
+ * @return {bool} `visible` If the source is visible.
```

Comment:
This whole project needs a non-passive rewrite for consistency.
I like using `visible` more than `render`, but I think they've been interchanged throughout.

Same out apply to `scale.x` and `cx`.

Reply:
> This whole project needs a non-passive rewrite for consistency.

Can't agree more. I also believe that other aspects must be rethinked and rewritten, which can be the basis for work on 5.0.0.

Reply:
@haganbmj maybe throw this in an issue for 5.0.0 cleanup